### PR TITLE
more 3d doc fixes

### PR DESCRIPTION
**Pull request by ritch**

### CODE REVIEW

File: docs/source/user_guide/groups.rst [hunk 1]
```diff
@@ -618,14 +618,16 @@ detections represented as |Detection| instances with their `label`, `location`,
     # Object label
     label = "vehicle"
 
-    # The center location ``(x, y, z)`` in camera coordinates of the bottom (y=0)
-    # face of the cuboid
+    # The center location of the object ``(x, y, z)`` in scene coordinates
+    # NOTE: the y coordinate will be offset by half the objects y size.
```
Comment (ritch) [inline docs/source/user_guide/groups.rst:622]: This is essentially a bug. I think we should just fix this via: https://github.com/voxel51/fiftyone/pull/2376

Comment (ritch) [reply docs/source/user_guide/groups.rst:622]: This was originally added to correct the positioning of cuboids in the kitti data set, but it should have been done via a setting, as the kitti labels are rotated.

Something like `settings['objectOriginStyle'] = 'centeredYFloor'`
or apply the offset to the labels themselves on import.

File: docs/source/user_guide/groups.rst [hunk 2]
```diff
@@ -1101,15 +1093,18 @@ shown below under the `plugins.3d` key of your
...
+                // The initial camera position in the 3D scene
                 "defaultCameraPosition": {"x": 0, "y": 0, "z": 0},
 
                 // Transformation from PCD -> reference coordinates
                 "pointCloud": {
                     // A rotation to apply to the PCD's coordinate system
                     "rotation": [0, 0, 0],
 
-                    // Don't render points below this z value
+                    // Define the lower z extent of the "color by height" range
```
Comment (brimoor) [inline docs/source/user_guide/groups.rst:1104]: To me, this first line makes it sound like points below `minZ` will still be rendered but will just have the same color as `minZ`. But later you clarify that this does indeed cull points below `minZ`.

Comment (ritch) [reply docs/source/user_guide/groups.rst:1107]: Ah that is confusing. Removing that line.

Comment (ritch) [reply docs/source/user_guide/groups.rst:1107]: See https://github.com/voxel51/fiftyone/pull/2379/files

**Comment by ritch** [comment]
Closing in favor of https://github.com/voxel51/fiftyone/pull/2379/files

**Pull request closed without merging**